test/e2e/helpers.js

```diff
@@ -39,6 +39,22 @@ const convertToHexValue = (val) => `0x${new BigNumber(val, 10).toString(16)}`;
 
 const convertETHToHexGwei = (eth) => convertToHexValue(eth * 10 ** 18);
 
+/**
```

Comment:
it was helpful to have proper types when refactoring to use the new `withWindowUntilClose` method, so I added them here (and fixed several usages of `withFixtures`).